### PR TITLE
Fix a typo

### PR DESCRIPTION
**Pull request by nichrosia**
Fixes a minor typo with the spore pod.

### CODE REVIEW

File: core/assets/bundles/bundle.properties [hunk 1]
```diff
@@ -1639,4 +1639,4 @@ lenum.mine = Mine at a position.
 lenum.build = Build a structure.
 lenum.getblock = Fetch a building and type at coordinates.\nUnit must be in range of position.\nSolid non-buildings will have the type [accent]@solid[].
 lenum.within = Check if unit is near a position.
-lenum.boost = Start/stop boosting.
\ No newline at end of file
+lenum.boost = Start/stop boosting.
```
Comment (joshuaptfan) [inline core/assets/bundles/bundle.properties:1642]: You inserted an unnecessary line break at the end of this line.

Comment (nichrosia) [reply core/assets/bundles/bundle.properties:1642]: This was done in the web editor, GitHub did that automatically. Any ideas on how to fix?

Comment (joshuaptfan) [reply core/assets/bundles/bundle.properties:1642]: Don't use the web editor to edit files.

Comment (nichrosia) [reply core/assets/bundles/bundle.properties:1642]: Sounds good.

Comment (Quezler) [reply core/assets/bundles/bundle.properties:1642]: #teamlinebreaks ❤️

**Comment by Anuken** [comment]
This is not a typo.

Add the implicit word "Spores" to the last sentence. Which is correct?

1. Spores **emit** gases toxic to other biological life.
2. Spores **emits** gases toxic to other biological life.

(answer: 1, the old text)

**Comment by nichrosia** [comment]
Oh. Forgot about this referencing plural objects.

**Pull request closed without merging**